ui/src/app/tasks-jobs/tasks/launch/free-text/free-text.component.html

```diff
@@ -28,23 +28,23 @@
             #propertiesFile
             (change)="propertiesFileChange($event)"
           />
-          Import a local file
+          {{ 'commons.importFile' | translate }}
         </button>
         <clr-tooltip-content clrPosition="top-right" clrSize="lg" *clrIfOpen>
-          <span
-            >Please provide a text file containing properties. This will be used to populate the text area above.</span
-          >
+          <span>{{ 'tasks.launch.freetext.propertiesTooltip' | translate }}</span>
         </clr-tooltip-content>
       </clr-tooltip>
 
-      <button id="btn-export" type="button" class="btn btn-sm btn btn-secondary" (click)="exportProps()">Export</button>
+      <button id="btn-export" type="button" class="btn btn-sm btn btn-secondary" (click)="exportProps()">
+        {{ 'tasks.launch.main.export' | translate }}
+      </button>
       <button id="btn-copy" type="button" class="btn btn-sm btn btn-secondary" (click)="copyPropsToClipboard()">
-        Copy to Clipboard
+        {{ 'tasks.launch.main.copy' | translate }}
```

Comment:
`Copy to Clipboard` is rendered on other sections.
Can we move these to `commons` translations ?